client/src/helpers/strings/platform.ts

```diff
@@ -58,13 +59,13 @@ export class PlatformStringProvider {
       this.strings = garantiCopy;
     } else if (platformType === PaymentPlatform.WISE) {
       this.strings = wiseCopy;
-    }  else {
+    } else {
       throw new Error('Invalid platform type');
     }
   }
 
   get(key: keyof PlatformStrings): string {
-    return this.strings[key];
+    return this.strings[key] ?? '';
```

Comment:
We should be even louder here i.e. raise an exception if an expected string is not returned